Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -73,6 +73,10 @@ shareRideInfo merchantId rideId = do
         DB.OneWayDetails locationDetail -> Just $ mkCommonBookingLocation locationDetail.toLocation
         DB.DriverOfferDetails driverOfferDetail -> Just $ mkCommonBookingLocation driverOfferDetail.toLocation
         _ -> Nothing
+  let mbDistance = case booking.bookingDetails of
+        DB.OneWayDetails locationDetail -> Just $ locationDetail.distance
+        DB.DriverOfferDetails driverOfferDetail -> Just $ driverOfferDetail.distance
+        _ -> Nothing
```

Comment:
in `OneWaySpecialZoneDetails` details we also have `distance` field

Reply:
Done